<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>417</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_editor">
       <attribute name="title">
        <string>Редактирование</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3dview">
       <attribute name="title">
        <string>Просмотр</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="OpenGLView" name="openGLView" native="true"/>
        </item>
        <item>
         <widget class="QGroupBox" name="colorEditBox">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>120</height>
           </size>
          </property>
          <property name="title">
           <string>Цвет фона</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="colorInputLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>R</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="redEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>G</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="greenEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>B</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="blueEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="updateButton">
             <property name="text">
              <string>Установить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpenGLView</class>
   <extends>QWidget</extends>
   <header>openglview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
